DOCUMENTATION.md

```diff
@@ -1431,10 +1431,10 @@ authentication feature which allows the Node.js clients also to have their priva
 to have their certificate authorities so that the members can know which clients they can trust. See the
 [Mutual Authentication section](#913-mutual-authentication).
 
-## 5.6. Enabling Hazelcast Cloud Discovery
+## 5.6. Enabling Hazelcast  Discovery
```

Comment:
```suggestion
## 5.6. Enabling Hazelcast Discovery
```

Also instead of deleting cloud, I think we should use

Enabling cloud discovery or Enabling Hazelcast Viridian discovery or Enabling Viridian Discovery

Reply:
This file hasn't changed, it was a Github glitch it was showing. I'm going to leave this as out of scope.